pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -122,6 +122,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	if kc.RegistryPullQPS < 0 {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: registryPullQPS (--registry-qps) %v must not be a negative number", kc.RegistryPullQPS))
 	}
+	if kc.MaxParallelImagePulls < 0 {
```

Comment:
I expected a non-nil MaxParallelImagePulls value to be required to be >= 2 (1 image pull is serial, not parallel)

I also expected `SerializeImagePulls: true` and non-nil MaxParallelImagePulls to be rejected in validation, not to ignore MaxParallelImagePulls

Reply:
makes sense. Will update